api/features/import_export/models.py

```diff
@@ -76,3 +76,13 @@ class FeatureImport(models.Model):
     # This is a JSON string of data generated by the export.
     data = models.CharField(max_length=MAX_FEATURE_IMPORT_SIZE)
     created_at = models.DateTimeField(auto_now_add=True)
+
+
+class FlagsmithOnFlagsmithFeatureExport(models.Model):
+    feature_export = models.ForeignKey(
+        FeatureExport,
+        related_name="flagsmith_on_flagsmith",
+        on_delete=models.CASCADE,
+    )
+
+    created_at = models.DateTimeField(auto_now_add=True)
```

Comment:
Interesting. I can certainly see why you went with this approach but I _think_ I'd prefer to just have an attribute on the FeatureExport class. Just thinking it's _slightly_ better aesthetics for the self hosted community where this functionality will be totally redundant. It'll be a redundant column vs a redundant table.

That being said, I don't have a massive preference here so if you think this is the cleanest approach, I'll take your word for it.

Reply:
I think it's better to keep the whole functionality limited to it's own class. I can see `FeatureExport` getting expanded and used more and more as the functionality evolves and every dev investigating it will see a field that is used almost none of the time and it will clutter up the definition of the `FeatureExport` class. I think it's better to just have a single class here which self-describes itself. Plus, if there are any other features we decide to implement in the future, like say hold onto different versions of flagsmith for download, we can add it to the `FlagsmithOnFlagsmithFeatureExport` class.

Reply:
Yep, fair enough. Happy with this approach in that case. Maybe we should add a docstring to the model however, to help explain it's purpose.

Reply:
Good idea I'll add it before merging the PR.